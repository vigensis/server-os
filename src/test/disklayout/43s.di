SCSI	disk0	MFG	SSD	100030242816	no	yes
SCSI	disk1	MFG	DISK	600127266816	no	no
SCSI	disk2	MFG	DISK	600127266816	no	no
SCSI	disk3	MFG	DISK	600127266816	no	no
SCSI	disk4	MFG	DISK	600127266816	no	no
SCSI	disk5	MFG	DISK	600127266816	no	no
SCSI	disk6	MFG	DISK	600127266816	no	no
SCSI	disk7	MFG	DISK	600127266816	no	no
SCSI	disk8	MFG	DISK	600127266816	no	no
SCSI	disk9	MFG	DISK	600127266816	no	no
SCSI	disk10	MFG	DISK	600127266816	no	no
SCSI	disk11	MFG	DISK	600127266816	no	no
SCSI	disk12	MFG	DISK	600127266816	no	no
SCSI	disk13	MFG	DISK	600127266816	no	no
SCSI	disk14	MFG	DISK	600127266816	no	no
SCSI	disk15	MFG	DISK	600127266816	no	no
SCSI	disk16	MFG	DISK	600127266816	no	no
SCSI	disk17	MFG	DISK	600127266816	no	no
SCSI	disk18	MFG	DISK	600127266816	no	no
SCSI	disk19	MFG	DISK	600127266816	no	no
SCSI	disk20	MFG	DISK	600127266816	no	no
SCSI	disk21	MFG	DISK	600127266816	no	no
SCSI	disk22	MFG	DISK	600127266816	no	no
SCSI	disk23	MFG	DISK	600127266816	no	no
SCSI	disk24	MFG	DISK	600127266816	no	no
SCSI	disk25	MFG	DISK	600127266816	no	no
SCSI	disk26	MFG	DISK	600127266816	no	no
SCSI	disk27	MFG	DISK	600127266816	no	no
SCSI	disk28	MFG	DISK	600127266816	no	no
SCSI	disk29	MFG	DISK	600127266816	no	no
SCSI	disk30	MFG	DISK	600127266816	no	no
SCSI	disk31	MFG	DISK	600127266816	no	no
SCSI	disk32	MFG	DISK	600127266816	no	no
SCSI	disk33	MFG	DISK	600127266816	no	no
SCSI	disk34	MFG	DISK	600127266816	no	no
SCSI	disk35	MFG	DISK	600127266816	no	no
SCSI	disk36	MFG	DISK	600127266816	no	no
SCSI	disk37	MFG	DISK	600127266816	no	no
SCSI	disk38	MFG	DISK	600127266816	no	no
SCSI	disk39	MFG	DISK	600127266816	no	no
SCSI	disk40	MFG	DISK	600127266816	no	no
SCSI	disk41	MFG	DISK	600127266816	no	no
SCSI	disk42	MFG	DISK	600127266816	no	no
SCSI	disk43	MFG	DISK	600127266816	no	no
USB	key0	USBMFG	USB Key	15737028608	yes	no
