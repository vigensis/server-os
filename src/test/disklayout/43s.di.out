{
	"spares": [
		{
			"name": "disk7",
			"vid": "MFG",
			"pid": "DISK",
			"size": "600127266816",
			"solid_state": false
		},
		{
			"name": "disk8",
			"vid": "MFG",
			"pid": "DISK",
			"size": "600127266816",
			"solid_state": false
		},
		{
			"name": "disk9",
			"vid": "MFG",
			"pid": "DISK",
			"size": "600127266816",
			"solid_state": false
		}
	],
	"vdevs": [
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk1",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk10",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk11",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk12",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk13",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk14",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk15",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk16",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk17",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk18",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk19",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk2",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk20",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk21",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk22",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk23",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk24",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk25",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk26",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk27",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk28",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk29",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk3",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk30",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk31",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk32",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk33",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk34",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk35",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk36",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk37",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk38",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk39",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk4",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk40",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk41",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk42",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk43",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk5",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "disk6",
					"vid": "MFG",
					"pid": "DISK",
					"size": "600127266816",
					"solid_state": false
				}
			]
		}
	],
	"capacity": 18003818004480,
	"logs": [
		{
			"name": "disk0",
			"vid": "MFG",
			"pid": "SSD",
			"size": "100030242816",
			"solid_state": true
		}
	]
}
